src/globals.cpp

```diff
@@ -224,6 +224,12 @@ std::queue<CTxInputData> txInQ GUARDED_BY(csTxInQ);
 // Transaction that cannot be processed in this round (may potentially conflict with other tx)
 std::queue<CTxInputData> txDeferQ GUARDED_BY(csTxInQ);
 
+// Transactions that arrive when the chain is not syncd can be place here at times when we've received
+// the block announcement but havn't yet downloaded the block and updated the tip. In this case there can
+// be txns that are perfectly valid yet are flagged as being non-final or has too many ancestors.
+std::queue<CTxInputData> txWaitQ GUARDED_BY(csTxInQ);
```

Comment:
"Wait" seems to be a bit ambiguous esp. given that we already have a "defer" queue.  How about txWaitNextBlockQ?  Or something better...

Reply:
done